components/CaseStudiesHero.tsx

```diff
@@ -0,0 +1,30 @@
+const heroText = ["Case", "Studies"];
```

Comment:
Move this inside its own index file along with any other dummy data or hard-coded data.